app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <table class="table mb-5">
+        <button type="button" class="btn btn-success mb-2 float-right"><%= link_to "Create Task", new_user_task_path, class:'text-light' %></button>
+          <thead class="bg-dark text-white">
+            <tr>
+              <th scope="col">Title</th>
+              <th scope="col">Description</th>
+              <th scope="col">Start Date</th>
+              <th scope="col">Actions</th>
+            </tr>
+          </thead>
+          <% @tasks.each do |task| %>
+          <tbody>
+            <tr>
+              <td><%= task.title %></td>
+              <td><%= task.description.truncate(28) %></td>
+              <td><%= formatted_date(task.started_at) %></td>
+              <td>
+                <button type="button" class="btn btn-success mb-1 btn-sm"><%= link_to "Edit", [:edit, current_user, task], class:'text-light' %></button>
```

Comment:
unnecessary button element and space issue in link_to

Reply:
done